File: docs/components_page/components/layout/vertical.R
library(dashBootstrapComponents)

row <- div(
  list(
    dbcRow(
      list(
        dbcCol(div("One of three columns")),
        dbcCol(div("One of three columns")),
        dbcCol(div("One of three columns"))
      ),
      align = "start"
    ),
    dbcRow(
      list(
        dbcCol(div("One of three columns")),
        dbcCol(div("One of three columns")),
        dbcCol(div("One of three columns"))
      ),
      align = "center"
    ),
    dbcRow(
      list(
        dbcCol(div("One of three columns")),
        dbcCol(div("One of three columns")),
        dbcCol(div("One of three columns"))
      ),
      align = "end"
    ),
    dbcRow(
      list(
        dbcCol(div("One of three columns"), align = "start"),
        dbcCol(div("One of three columns"), align = "center"),
        dbcCol(div("One of three columns"), align = "end")
      )
    )
  ),
  className = "pad-row"
)

File: docs/components_page/components/layout/vertical.jl
using DashBootstrapComponents, DashHtmlComponents

row = html_div([
    dbc_row(
        [
            dbc_col(html_div("One of three columns")),
            dbc_col(html_div("One of three columns")),
            dbc_col(html_div("One of three columns")),
        ],
        align = "start",
    ),
    dbc_row(
        [
            dbc_col(html_div("One of three columns")),
            dbc_col(html_div("One of three columns")),
            dbc_col(html_div("One of three columns")),
        ],
        align = "center",
    ),
    dbc_row(
        [
            dbc_col(html_div("One of three columns")),
            dbc_col(html_div("One of three columns")),
            dbc_col(html_div("One of three columns")),
        ],
        align = "end",
    ),
    dbc_row([
        dbc_col(html_div("One of three columns"), align = "start"),
        dbc_col(html_div("One of three columns"), align = "center"),
        dbc_col(html_div("One of three columns"), align = "end"),
    ]),
]);
